### ISSUE TITLE
Can't activate air Conditioning over UI

### ISSUE THREAD

**Issue by kotarokun** [skodaconnect/homeassistant-myskoda#42]
Hello,
I want to change Air conditioning via UI but it always stand on auto. So I can only switch off if it is heating.
But I can switch via climate.turn_on and it works. 

Maybe just an switch to the actually state? So it stand off and can switch to auto?

Thanks for your help

**Comment by WebSpider** [comment]
What version of HA and what version of the integration are you running?

**Comment by WebSpider** [comment]
Also, are there any additional parameters you pass to `climate.turn_on`?

**Comment by kotarokun** [comment]
I use actually 2024.06.4
And climate.turn_on works an I see in app it will send and after a while it will preheating the car.
I just add the Enyaq climate-entity. W
I be home in two days and maybe I have a look before

**Comment by WebSpider** [comment]
Thank you. I will check on 2024.06 and see if i can reproduce this.

**Comment by lightheaded** [comment]
For reference, I have two cars - Skoda Superb and Nissan Leaf. Nissan's climate works as expected, but I can't turn on my Škoda's climate entity via the UI, as it's seemingly always "on". I'm using latest HA and the addon.

In the actual "off" state:

Skoda:
![image](https://github.com/user-attachments/assets/69577533-ca97-4a88-840c-023101c8b23f)
![image](https://github.com/user-attachments/assets/518a6657-35b8-4135-8d64-684e20fc3dff)

Nissan:
![image](https://github.com/user-attachments/assets/77a1122e-d4ba-4cfe-b4d0-76421bbb006e)
![image](https://github.com/user-attachments/assets/d68d3dd4-fc5d-4d2f-b28a-851e190e97ec)

And in "on" state (i.e. climate manually started from the app):

Skoda:
![image](https://github.com/user-attachments/assets/ef9f21e5-e6f9-46bd-8ed9-b8347b46c3e8)
![image](https://github.com/user-attachments/assets/204b4290-b8f5-4939-97d5-54656fa046c6)

Nissan:
![image](https://github.com/user-attachments/assets/82a2f86e-1743-4dc8-9082-78633d708805)
![image](https://github.com/user-attachments/assets/b9574ec4-31c8-4468-8f08-d8ec8fbaddac)

**Comment by lightheaded** [comment]
I am using this script as a workaround, as it toggles based climate on the `hvac_action` attribute:

```yaml
alias: Toggle Skoda Climate
sequence:
  - choose:
      - conditions:
          - condition: state
            entity_id: climate.skoda_superb_air_conditioning
            attribute: hvac_action
            state: "off"
        sequence:
          - action: climate.turn_on
            metadata: {}
            data: {}
            target:
              entity_id: climate.skoda_superb_air_conditioning
      - conditions:
          - condition: not
            conditions:
              - condition: state
                entity_id: climate.skoda_superb_air_conditioning
                attribute: hvac_action
                state: "off"
        sequence:
          - action: climate.turn_off
            metadata: {}
            data: {}
            target:
              entity_id: climate.skoda_superb_air_conditioning
description: ""
icon: mdi:car-defrost-front
```

And then in the card:

```yaml
          - type: custom:mushroom-climate-card
            entity: climate.skoda_superb_air_conditioning
            icon: ""
            layout: horizontal
            fill_container: false
            secondary_info: last-changed
            show_temperature_control: false
            hvac_modes: []
            primary_info: name
            name: Škoda Climate
            tap_action:
              action: perform-action
              perform_action: script.toggle_skoda_climate
              target: {}
```

**Comment by WebSpider** [comment]
Thanks, I need to dive into the inner workings of climate entities a bit and see if I can fix this. In the mean time: PR's are welcome!

**Comment by WebSpider** [comment]
This has been fixed in v1.6.2
